changelog/SLgUAncsTtS1C3Vbv7vYyA.md

```diff
@@ -0,0 +1,4 @@
+audience: developers
+level: patch
+---
+The Go implementation of the runner / worker protocol is now an internal library and not accessible from outside the Tasksluter repository.
```

Comment:
one day we will spell this correctly

Reply:
One more misspelling to add to the meta tests :)